packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.tsx

```diff
@@ -49,11 +50,17 @@ export class MessageBar extends React.Component<IMessageBarProps, IMessageBarSta
     /**
      * Live regions need an update to announce content.
      */
-    setTimeout(() => {
+    this._mountTimeout = setTimeout(() => {
```

Comment:
Yes, I am aware of that but I was not inclined to put new dependencies and add another class hierarchy just to reuse a small code. I would rather take that dependency for something involving throttling/debounce. I think we should strive to keep components dependencies small until unless really required. Thoughts ?